### DISCUSSION TITLE
[ADD] A non-project specific addons directory.

### DISCUSSION THREAD

**Discussion by VerzatileDev** [godotengine/godot-proposals#8901] [General Discussions]
Platform
---

Windows 10
Godot 4.2.1 Stable

Proposal
---

When downloading certain Addons such as of trackers, Ui/gui changers and many alike, there should be an option to specific Addons that are not specific to the Project itself, but rather the engine itself.

Example
---

As an example a popular Addon: https://godotengine.org/asset-library/asset/236 of Wakatime, a time tracker of the engine managed and used, should be specific to the engine not the project.

As of now there are work-arounds in changing the directory, but are not specifically the best solution as referenced here: https://github.com/godotengine/godot-proposals/issues/554 
, where additionally the use of a package manager to automatically download the Addons.

In 2018, there was an issue related referenced about the use of Hardcoding of the Addons, that third-party Addons such of plugins / tools etc, should still be referenced there, eventhough they are not of " Editor specific plugins ". https://github.com/godotengine/godot/issues/17007#issuecomment-646003253

- Meaning that the editor, building of the project and for the project to work does not specifically require that specific plugin.

Solves
---

The following would solve the need to manually add the extentsion for each project and the assigning of the key.

Plugins such as (Wakatime) do not require to be built to the release and builds.

Removes the need to use git ignore for version control specific to non editor specific plugins.

**Comment by AThousandShips** [comment]
Sounds like a duplicate of:
* https://github.com/godotengine/godot-proposals/issues/831

**Reply by VerzatileDev** [reply]
Apologies, seems that I had missed discussion of #831 , appriciate it!

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/831.
